src/app/team/page.tsx

```diff
@@ -1,5 +1,100 @@
+import ContentContainer from "../../../components/ui/ContentContainer";
+import Link from "next/link";
+
 export default function page() {
   return (
-    <div>page</div>
+    <div>
+      {/* Title block */}
+      <div className="w-full h-40 flex items-center pl-6 bg-secondary-blue">
+        <h1 className="mb-2 mt-0 text-3xl font-medium leading-6 sm:ml-12 text-primary">Meet Our Team</h1>
+      </div>
+      {/* 2 lateral content containers*/}
+      <div className="bg-secondary-blue flex flex-col sm:flex-row justify-evenly gap-4 px-4 " >
+        <ContentContainer>
+          <div>
+            <p className="text-base lg:text-2xl font-medium">Project Director</p>
+            <p className="text-base lg:text-lg">Ashfaq Ishaq, Ph.D.
+              <Link href="https://www.linkedin.com/in/ashfaqishaq/" className="inline-block ml-3">
+                <img src="/team/icon-linkedin.svg" alt="LinkedIn company logo" className="h-6 w-6" />
+              </Link>
+            </p>
+            <p className="text-base lg:text-2xl font-medium">Director, Community Relations</p>
+            <p className="text-base lg:text-lg">Katty Guerami
+              <Link href="https://www.linkedin.com/in/katty-guerami-74a99014/" className="inline-block ml-3">
+                <img src="/team/icon-linkedin.svg" alt="LinkedIn company logo" className="h-6 w-6" />
+              </Link>
+            </p>
+            <p className="text-base lg:text-2xl font-medium">ICAF Paris</p>
+            <p className="text-base lg:text-lg">Katherine Harold, Paris College of Art
+              <Link href="https://www.linkedin.com/in/katherine-harold-23a5a0137/" className="inline-block ml-3">
+                <img src="/team/icon-linkedin.svg" alt="LinkedIn company logo" className="h-6 w-6" />
+              </Link>
+            </p>
+          </div>
+        </ContentContainer>
+        <ContentContainer>
+          <div>
+            <p className="text-base lg:text-2xl font-medium">Legal Advisors</p>
+            <p className="text-base lg:text-lg">Oroma Womeodu, Esq.</p>
+            <p className="text-base lg:text-lg">Jade Monkouopm</p>
```

Comment:
Screen reader users will be pleased if content visually represented as a list is represented as such using `ul`/`li`.

Reply:
Thanks for all the feedback, I believe I have implemented the fixes for all of it.